CorsixTH/Lua/config_finder.lua

```diff
@@ -125,6 +125,7 @@ local config_defaults = {
   enable_avg_contents = false,
   remove_destroyed_rooms = false,
   machine_menu_button = true,
+  enable_screen_shake = true,
```

Comment:
You'll need to add the text you made in the config template also here in the config finder. You can mostly copy paste an existing setting and adapt it as needed.